bats/tests/snapshots/test_rdctl_snapshot.bats

```diff
@@ -0,0 +1,64 @@
+load '../helpers/load'
```

Comment:
IMO this file is doing things that unit tests are better suited to. In fact, most of the test cases are already covered by unit tests (see `src/go/rdctl/pkg/snapshot/manager_test.go`). For the ones that aren't, it would be better to just add unit tests for them.

Reply:
Yup, I reduced it down to one test, that runs `rdctl` -- that layer is hard to unit-test without mocking the cobra code. I'd rather not.